firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Unreleased
+- [fixed] Fixed an issue that prevented schema migrations for clients with
+  large offline datasets (#370, #734).
```

Comment:
What does 734 refer to?  https://github.com/firebase/firebase-android-sdk/issues/734 is a 404

Maybe you meant 374, which is this PR, but I'm not sure if that's particularly helpful?  Presumably this PR will be associated with issue 370 anyway, so folks can find their way to it easily enough if they really want to dig into it?

Reply:
I removed the link to the PR. In general, I have been linking to both in changelogs, but the more common precedent in other SKDs seems to be to link to neither. One links seems to be a good comprise :)